engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -119,7 +129,7 @@ object Runtime {
     @named("expressionUpdate")
     case class ExpressionUpdate(
       expressionId: ExpressionId,
-      expressionTypes: Option[Vector[String]],
+      expressionType: Option[ExpressionType],
```

Comment:
This is (again) an incompatible change in the protocol. Shouldn't we rather start to be compatible in the absence of #7717? What about:
- keeping `expressionTypes` unchanged
- adding optional `dispatchTypes` with `publicTypes = typeOfNode.findAllTypesOrNull(value, false);`
- note that when `dispatchTypes` are missing, then the first type of `expressionTypes` is _dispatchable_

Reply:
It is an internal API between the runtime and the language server. We cannot run them separately, i.e. different versions